### PR TITLE
Updated the Ray image in docs/nbs to 2.20.0

### PR DESCRIPTION
**Pull request by Bobbins228**
# Issue link

[RHOAIENG-6450](https://issues.redhat.com/browse/RHOAIENG-6450)
# What changes have been made

Updated all doc & notebook images to reflect new ray image
# Verification steps

## Checks
- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [ ] Testing is not required for this change

### CODE REVIEW

File: demo-notebooks/guided-demos/0_basic_ray.ipynb [hunk 1]
```diff
@@ -53,24 +53,32 @@
...
+     ]
+    }
+   ],
    "source": [
     "# Create and configure our cluster object\n",
     "# The SDK will try to find the name of your default local queue based on the annotation \"kueue.x-k8s.io/default-queue\": \"true\" unless you specify the local queue manually below\n",
     "cluster = Cluster(ClusterConfiguration(\n",
     "    name='raytest', \n",
-    "    namespace='default', # Update to your namespace\n",
+    "    namespace='mark-dsp', # Update to your namespace\n",
```
Comment (KPostOffice) [inline demo-notebooks/guided-demos/0_basic_ray.ipynb:73]: Testing artifact

File: demo-notebooks/guided-demos/0_basic_ray.ipynb [hunk 2]
```diff
@@ -53,24 +53,32 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
```
Comment (KPostOffice) [inline demo-notebooks/guided-demos/0_basic_ray.ipynb:56]: Do we care about these remaining `null`? Should be solveable by just restarting the notebook before committing changes?

File: demo-notebooks/guided-demos/1_cluster_job_client.ipynb [hunk 3]
```diff
@@ -218,7 +218,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.18"
+   "version": "3.9.19"
```
Comment (KPostOffice) [inline demo-notebooks/guided-demos/1_cluster_job_client.ipynb:221]: There are still a few example notebooks with `3.9.19` can you fix them as well in this PR?

File: demo-notebooks/guided-demos/0_basic_ray.ipynb (hunk 2 above)
Comment (Bobbins228) [reply demo-notebooks/guided-demos/0_basic_ray.ipynb]: We should keep them null just for the sake of consistency :+1:

**Review by KPostOffice — APPROVED** [review]
/lgtm

**Pull request merged**